Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -435,69 +442,66 @@ public void indexConcepts(String userName)
+                    analyzerPerField.put(luceneFieldAnnotation.lucenefieldName(), whiteSpaceAnalyzer);
+                } else {
+                    // Always analyze the short field name with whitespace
+                    // for non tokenized.
+                    analyzerPerField.put(luceneFieldAnnotation.luceneShortFieldName(), whiteSpaceAnalyzer);
+                }
+            }
+
+        }
+
+        PerFieldAnalyzerWrapper perFieldAnalyzerWrapper = new PerFieldAnalyzerWrapper(standardAnalyzer,
+                analyzerPerField);
+
+        QueryBuilder qBuild = new QueryBuilder(perFieldAnalyzerWrapper);
+        BooleanQuery.Builder builder = new BooleanQuery.Builder();
+
         for (java.lang.reflect.Field field : fields) {
             SearchField search = field.getAnnotation(SearchField.class);
             LuceneField luceneFieldAnnotation = field.getAnnotation(LuceneField.class);
             if (search != null) {
                 String searchString = fieldMap.get(search.fieldName());
-
-                if(searchString != null){
-                    if (firstEntry != 1)
-                        queryString.append(" " + operator + " ");
-                    firstEntry++;
-                    queryString.append(luceneFieldAnnotation.lucenefieldName() + ":");
-
-                    StringBuffer searchBuffer = new StringBuffer("(");
-                    String[] searchParts = searchString.split(" ");
-                    
-                    boolean quoteOpen = false;
-                    for (String term : searchParts) {
-                        if (!quoteOpen && !term.trim().isEmpty()) {
-                            searchBuffer.append("+");
-                        }
-                        
-                        searchBuffer.append(QueryParser.escape(term) + " ");
-                        
-                        if (term.startsWith("\"")) {
-                            quoteOpen = true;
-                        }
-                        
-                        if (term.endsWith("\"")) {
-                            quoteOpen = false;
+                if (searchString != null) {
+                    if (!luceneFieldAnnotation.isShortPhraseSearchable()) {
+                        Query q = qBuild.createPhraseQuery(luceneFieldAnnotation.lucenefieldName(), searchString);
+                        builder.add(q, occur);
+                    } else {
+                        QueryBuilder shortTermsQueryBuilder = new QueryBuilder(perFieldAnalyzerWrapper);
+                        BooleanQuery.Builder shortTermBooleanQueryBuilder = new BooleanQuery.Builder();
+                        Query q1 = shortTermsQueryBuilder.createPhraseQuery(luceneFieldAnnotation.lucenefieldName(),
+                                searchString);
+                        Query q2 = shortTermsQueryBuilder.createBooleanQuery(
+                                luceneFieldAnnotation.luceneShortFieldName(), searchString.replaceAll(" ", ""));
```

Comment:
how about instead of using an extra attribute of the annotation to define a field name, just make the field name: ```luceneFieldName() + "_untokenized"```? ```luceneShortFieldName``` is really not self-explanatory and just one more thing to configure